nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/commands/IndexingCommandsStacker.java

```diff
@@ -154,6 +158,10 @@ protected void stackCommand(DocumentModel doc, String eventId, boolean sync) {
         }
     }
 
+    private boolean indexIsLatestVersion() {
```

Comment:
Reverse the logic and call it `isLatestVersionDisabled` for consistency with code in `DBSSession`.